config-controller/main.go

```diff
@@ -124,8 +127,14 @@ func main() {
 		Metrics:                metricsServerOptions,
 		WebhookServer:          webhookServer,
 		HealthProbeBindAddress: probeAddr,
-		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "8575d0a9.stackrox.io",
+		NewCache: func(config *rest.Config, opts cache.Options) (cache.Cache, error) {
+			opts.DefaultNamespaces = map[string]cache.Config{
+				env.Namespace.Setting(): {},
+			}
+			return cache.New(config, opts)
+		},
+		LeaderElection:   enableLeaderElection,
```

Comment:
Should we be removing these since we will have only one replica?

Reply:
We can remove it, but it doesn't hurt leaving it around since the default is to disable when the `--leader-elect` flag is omitted.

Reply:
4f03b489dbb317015674a0bb0b1b2a1d4bf9a877